### PR TITLE
[pt_BR] Update locales/pt_BR/pt_BR.json

### PR DESCRIPTION
**Pull request by VerasNp**
Address
City
Country

### CODE REVIEW

File: locales/pt_BR/pt_BR.json [hunk 1]
```diff
@@ -441,7 +441,7 @@
     "Please provide your name.": "Por favor, forneça o seu nome.",
     "Poland": "Polônia",
     "Portugal": "Portugal",
-    "Press \/ to search": "Pressione \/ para pesquisar",
+    "Press / to search": "Pressione / para pesquisar",
```
[the remark this answers is not in the document]
Comment (VerasNp) [reply locales/pt_BR/pt_BR.json]: I think now its fine, can you check please?

**Comment by caouecs** [comment]
Obrigado

**Comment by caouecs** [comment]
Can you help for the rest of translations ? https://laravel-lang.github.io/lang/statuses/pt-br.html

Thanks by advance

**Comment by VerasNp** [comment]
Yep, this repository helps me alot.

**Pull request merged**